build/images/build-utils.sh

```diff
@@ -41,3 +41,56 @@ function check_docker_build_driver() {
     fi
     return 0
 }
+
+function switch_windows_buildx() {
+    local windows_buildx_name="windows-img-builder"
+    original_buildx_instance=$(docker buildx inspect | grep -oP '^Name:\s*\K\S+' | head -n 1)
```

Comment:
please use a command that will work on macOS as well
maybe a command similar to the one in `get_docker_build_driver`?

Reply:
updated